packages/core/src/api/logs/initialize.test.ts

```diff
@@ -167,4 +171,54 @@ describe('api.logs', () => {
       });
     });
   });
+
+  describe('User action', () => {
+    it('buffers the error if a user action is in progress', () => {
```

Comment:
what error is being buffered here? 

should this say:
```suggestion
    it('buffers the item if a user action is in progress', () => {
```